library/src/main/java/com/novoda/downloadmanager/lib/BatchRepository.java

```diff
@@ -147,7 +141,7 @@ public DownloadBatch retrieveBatchFor(DownloadInfo download) {
     }
 
     public List<DownloadBatch> retrieveBatchesFor(Collection<DownloadInfo> downloads) {
-        Cursor batchesCursor = resolver.query(batchContentUri, null, null, null, null);
+        Cursor batchesCursor = resolver.query(this.downloads.getBatchContentUri(), null, null, null, null);
```

Comment:
Yup because downloads is both a param and a class field. That's why when the `Downloads` class is renamed to `UriProvider` we won't need to do that